src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml

```diff
@@ -0,0 +1,118 @@
+
+                        <ul class="govuk-list govuk-list--bullet">
+                            <li>view training provider contact details</li>
+                            <li>cancel your request</li>
+                        </ul>
+                        <p>Requests will expire after @Model.Dashboard.ExpiryAfterMonths months.</p>
+                        <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-top-6 govuk-!-margin-bottom-6" />
+                        foreach(var request in Model.Dashboard.AggregatedEmployerRequests)
+                        {
+                            if (request.NewNumberOfResponses > 0)
+                            {
+                                <strong class="govuk-tag">
+                                    New
+                                    @if (request.NewNumberOfResponses == 1)
+                                    {
+                                        <text>response</text>
+                                    }
+                                    else
+                                    {
+                                        <text>responses</text>
+                                    }
+                                </strong>
+                            }
+                            <p>
+                                <div class="govuk-body govuk-!-margin-bottom-1">
+                                    <a 
+                                        asp-route="@EmployerRequestController.ViewProviderResponsesRouteGet"
+                                        asp-route-hashedAccountId="@Model.HashedAccountId"
+                                        asp-route-employerRequestId="@request.EmployerRequestId" 
+                                        class="govuk-link govuk-link--no-visted-state">@request.StandardTitle (level @request.StandardLevel)
+                                    </a>
+                                </div>
+                                <div class="govuk-hint">
+                                    @request.NumberOfResponses
+                                    @if (request.NumberOfResponses == 1)
+                                    {
+                                        <text>response</text>
+                                    }
+                                    else
+                                    {
+                                        <text>responses</text>
+                                    }
+                                </div>
+                            </p>
+                            <p>
+                                <div class="govuk-body govuk-!-margin-bottom-1">Request made on @request.RequestedAt.ToString("d MMM yyyy")</div>
+                                <div class="govuk-hint">
+                                    Expires in @((request.ExpiryAt - DateTime.Now.Date).Days) days (@request.ExpiryAt.ToString("d MMM yyyy"))
```

Comment:
If we are using reponse/responses above, should we also differentiate between day/days.  i don't think this is mentioned consistently in the user stories though

Reply:
AC updated will make change